### ISSUE TITLE
Sentry\Monolog\Handler: Support logging of $context

### ISSUE THREAD

**Issue by glensc** [getsentry/sentry-php#1270]
The monolog handler doesn't save $context parameter and seems there's no way to customize that:
- https://github.com/getsentry/sentry-php/blob/ecbd09ea5d053a202cf773cb24ab28af820831bd/src/Monolog/Handler.php#L40-L62

**Comment by glensc** [comment]
```php
             $scope->setExtra('monolog.context', $record['context']); 
```

this would send everything

**Comment by ste93cry** [comment]
Thank you for the contribution. I already started the work in #1244, and although it has been abandoned for some time I intend to get back to it in the new year. Therefore, I'm closing this PR as duplicate

**Comment by ste93cry** [comment]
This has been fixed in #1244 and will be released with the next minor version
